doc/ug/conf.py

```diff
@@ -12,6 +12,16 @@
 # serve to show the default.
 
 import sys, os
+import datetime
+
+# Gets version directly from the code
+try:
+    import espressopp
```

Comment:
Not sure if that will even use the current version or the one which is installed.
I might be better to turn `conf.py` into a cmake configure_file and replace the version at cmake time.